Makefile

```diff
@@ -1,7 +1,9 @@
 # Operator variables
 # ==================
 export APP_NAME=compliance-operator
+RESULTCOLLECTORBIN=resultcollector
 OPENSCAP_IMAGE_NAME=openscap-ocp
+RESULTSCOLLECTOR_IMAGE_NAME=$(RESULTCOLLECTORBIN)
```

Comment:
having both RESULTCOLLECTOR and RESULTSCOLLECTOR var names makes it a little hard to read, lets use one of them for all related vars